### ISSUE TITLE
Question about the setting of period_base_list

### ISSUE THREAD

**Issue by Carkham** [KevinZhoutianyi/FoNE#2]
Hi, thanks for this amazing works!

I would like to inqure is there any suggestion on how to choose a good value for `period_base_list`? For example, if I only need to handle numbers between 0 and 100, can I just set `period_base_list=[20, 50]` and `logits.view(-1, 100)` for prediciton?

**Comment by KevinZhoutianyi** [comment]
Sorry for the late reply!! There is no email notification.

For high precision you can use [2,5,20,50], as given mod 2 and mod 5 you know the unit digit and given mod 20 and mod 50 you know the ten digit.

Or you can use [10,100]. From my experience, it will achieve similar performance.
